### PR TITLE
Remove "obviously" disabled resources (SOFTWARE-3351)

### PR DESCRIPTION
**Pull request by edquist**

### CODE REVIEW

File: topology/University of California Santa Cruz/UC Santa Cruz ATLAS Tier-3/UCSCT3.yaml [hunk 1]
```diff
@@ -1,34 +0,0 @@
-Disable: true
-GroupDescription: |-
-  Resource for UC Santa Cruz ATLAS Tier-3.
-
-  7/26 disabling empty resource group
-GroupID: 291
-Production: true
-Resources:
-  UCSC_ATLAS_GRIDFTP2:
```
[the remark this answers is not in the document]
Comment (brianhlin) [reply topology/University of California Santa Cruz/UC Santa Cruz ATLAS Tier-3/UCSCT3.yaml]: @efajardo ping

[the remark this answers is not in the document]
Comment (efajardo) [reply topology/University of California Santa Cruz/UC Santa Cruz ATLAS Tier-3/UCSCT3.yaml]: @brianhlin I do not know. I think someone like Doug Benjamin would be better to ask

[the remark this answers is not in the document]
Comment (brianhlin) [reply topology/University of California Santa Cruz/UC Santa Cruz ATLAS Tier-3/UCSCT3.yaml]: @edquist Let's revive this resource (Jason verified that it's still up and doing ATLAS work)

**Comment by edquist** [comment]
Updated

**Pull request merged**